iaso/models/base.py

```diff
@@ -1817,6 +1827,9 @@ class FeatureFlag(models.Model):
     name = models.CharField(max_length=100, null=False, blank=False)
     requires_authentication = models.BooleanField(default=False)
     description = models.TextField(blank=True)
+    category = models.TextField(choices=FeatureFlagCategory.choices, default=FeatureFlagCategory.NA)
+    order = models.IntegerField(default=0)
```

Comment:
Nitpick: I don't think it makes a big difference, but I would have used
```suggestion
    order = models.PositiveSmallIntegerField(default=0)
```